libs/src/services/ethContracts/EthContracts.ts

```diff
@@ -11,34 +11,24 @@ import { ClaimDistributionClient } from './ClaimDistributionClient'
 import { WormholeClient } from './WormholeClient'
 import { EthRewardsManagerClient } from './EthRewardsManagerClient'
 import { TrustedNotifierManagerClient } from './TrustedNotifierManagerClient'
-import { Logger, Utils } from '../../utils'
+import type { Logger } from '../../utils'
 import type { EthWeb3Manager } from '../ethWeb3Manager'
 import type { ContractClient } from '../contracts/ContractClient'
 
-const AudiusTokenABI = Utils.importEthContractABI('AudiusToken.json').abi
-const RegistryABI = Utils.importEthContractABI('Registry.json').abi
-const GovernanceABI = Utils.importEthContractABI('Governance.json').abi
-const ServiceTypeManagerABI = Utils.importEthContractABI(
-  'ServiceTypeManager.json'
-).abi
-const ServiceProviderFactoryABI = Utils.importEthContractABI(
-  'ServiceProviderFactory.json'
-).abi
-const StakingABI = Utils.importEthContractABI('Staking.json').abi
-const DelegateManagerABI = Utils.importEthContractABI(
-  'DelegateManagerV2.json'
-).abi
-const ClaimsManagerABI = Utils.importEthContractABI('ClaimsManager.json').abi
-const ClaimDistributionABI = Utils.importEthContractABI(
-  'AudiusClaimDistributor.json'
-).abi
-const WormholeClientABI = Utils.importEthContractABI('WormholeClient.json').abi
-const EthRewardsManagerABI = Utils.importEthContractABI(
-  'EthRewardsManager.json'
-).abi
-const TrustedNotifierManagerABI = Utils.importEthContractABI(
-  'TrustedNotifierManager.json'
-).abi
+import { abi as AudiusTokenABI } from '../../eth-contracts/ABIs/AudiusToken.json'
```

Comment:
So we are no longer lazy loading these at all?

Reply:
yeah exactly, i think that's the move long term since the files aren't big and it wasn't working for native anyway. you feel cool with it?